src/commands/new.rs

```diff
@@ -1,14 +1,120 @@
-use anyhow::{Context, Result};
+use anyhow::{bail, Context, Result};
 use cargo::{
     ops::{self, NewOptions, VersionControl},
     Config,
 };
 use clap::{ArgAction, Args};
-use std::{fs, path::Path};
+use heck::{ToKebabCase, ToSnakeCase, ToUpperCamelCase};
+use std::{
+    borrow::Cow,
+    fmt, fs,
+    path::{Path, PathBuf},
+};
 use toml_edit::{table, value, Document, InlineTable, Item, Table, Value};
 
 use crate::WIT_BINDGEN_REPO;
 
+fn is_wit_keyword(s: &str) -> bool {
+    // TODO: move this into wit-parser?
```

Comment:
I think this would be generally useful for other languages so moving to wit-parser makes sense. Shouldn't block getting this change in however

Reply:
Agreed.

Ideally the Rust one won't be necessary, either. Currently wit-bindgen-rust doesn't generate raw identifiers for any wit identifiers that are Rust keywords, so I figured we'd preemptively block them here and remove the restriction once it does.